lms/djangoapps/mobile_api/testutils.py

```diff
@@ -44,7 +44,7 @@ def setUp(self):
         self.course = CourseFactory.create(
             mobile_available=True,
             static_asset_path="needed_for_split",
-            end=datetime.datetime.now(pytz.UTC),
+            end=datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=1),
```

Comment:
@shafqatfarhan  Do we need to add a comment here?

Reply:
@ihtram Comment added